src/braket/ir/jaqcd/instructions.py

```diff
@@ -899,3 +900,40 @@ class Type(str, Enum):
         kraus = "kraus"
 
     type = Type.kraus
+
+    """
+    StartPreserveBLock.
```

Comment:
Please provide a description of the functionality.

Reply:
Updated